### ISSUE TITLE
Update runtime org.kde.Platform

### ISSUE THREAD

**Issue by Shilionz** [flathub/br.eng.silas.qpdftools#4]
running `flatpak update` gives the following warning: 
```
Info: runtime org.kde.Platform branch 6.6 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Platform and SDK
Info: applications using this runtime:
   br.eng.silas.qpdftools
```
could you please check if this is an issue? 
Thank you very much for your effort on this project!

**Comment by silash35** [comment]
Yes, this was an issue. My fault, I should have updated to KDE Runtime 6.8 earlier, as soon as 6.6 was approaching end-of-life.

For some reason, I wasn't receiving notifications from this repository, so I hadn't seen your issue or @PunkPangolin's pull request. Sorry for the delay, but everything is fixed now.
